src/shared/utils/functions.tsx

```diff
@@ -8,6 +9,23 @@ export const computePokemonNumber = (pokemonId: number) => {
   return '#'+ ('000'+pokemonId.toString()).slice(-3); 
 };
 
+const generateRandomLocation = () => {
+  const latitude = telAvivBounds.minLat + Math.random() * (telAvivBounds.maxLat - telAvivBounds.minLat);
+  const longitude = telAvivBounds.minLng + Math.random() * (telAvivBounds.maxLng - telAvivBounds.minLng);
+  return { latitude: latitude, longitude: longitude}  as PokemonLocation;
+};
+
+export const getPokemonLocation = (pokemonDataId: number) => {
+  const storedLocation = localStorage.getItem(`pokemonLocation_${pokemonDataId}`);
```

Comment:
for both appearances of `pokemonLocation_${pokemonDataId}`, i would have make more generic so if, in the future, i would want to change the key for the localstorage data, it would be in one go. i can be as a function that generate the string or any other way :)